### ISSUE TITLE
New script setup allow "dot" custom component import and name tag

### ISSUE THREAD

**Issue by KaliaJS** [vuejs/vue-syntax-highlight#211]
The new SFC <script setup> allow to import all customs components and use dot notation on the template.

```javascript
// Example
<script setup>
import * as Ui from '@plugins/vue-ui'
</script>

<template>
  <Ui.Content></Ui.Content>
</template>
```

The problem is that everything after the dot is "Tag attribute"  and not "Tag name" (sublime text 4). Can something be done with vue-syntax-highlight ?

![image](https://i.imgur.com/KSKK22W.png)

**Comment by rchl** [comment]
I don't see that behavior here.

![Screenshot 2021-09-09 at 22 00 55](https://user-images.githubusercontent.com/153197/132754603-1dbdb726-fbff-460c-ad5a-90e7c3209d35.png)

**Comment by rchl** [comment]
Actually it seems like it was fixed recently for ST4 (through #209)

**Comment by KaliaJS** [comment]
@rchl You're right thank you
